forum/investors/views.py

```diff
@@ -1,3 +1,12 @@
-from django.shortcuts import render
+from django.shortcuts import render, HttpResponse
```

Comment:
You import django.shortcuts render but unused it

Reply:
Left unused import as a placeholder for next development. When you create a new app, view contain _
from django.shortcuts import render_ by default.